UserAgent/Views/Privacy Indicator/PrivacyDashboardUtils.swift

```diff
@@ -0,0 +1,139 @@
+import Shared
```

Comment:
This is definitely not in scope of this PR, but it got me thinking. If we are going to encapsulate all Views in own framework/project that probably we should try to avoid all imports. That would require us to pass translations as arguments to the views. It should not be that much of a deal, but would like to see how it would look like.

Reply:
[just thoughts]
I think there are two options: 1) import (like we do in nav-ext if i am not wrong) 2) pass a props; 
Second option better, drawback we need to pass props to innermost component or create sort of implicit context. I think second option much more reusable but it comes with some (small) cost